flink-runtime/pom.xml

```diff
@@ -117,6 +117,16 @@ under the License.
 		</dependency>
 
 		<dependency>
+			<groupId>org.slf4j</groupId>
+			<artifactId>slf4j-log4j12</artifactId>
+		</dependency>
+
+		<dependency>
+			<groupId>log4j</groupId>
```

Comment:
IIRC we explicitly removed these dependencies so that other logging frameworks can be plugged in.